README.rst

```diff
@@ -19,6 +19,9 @@ dwave-neal
 .. index-start-marker
 
 An implementation of a simulated annealing sampler.
+A simulated annealing sampler can be used for approximate Boltzmann sampling, or heuristic optimization. In this implementation the equilibrium distribution at a target inverse temperature (beta) is approached by performing updates at a sequence of increasing beta values (the beta_schedule), terminating at the target beta. In this implementation each spin is updated once in a fixed order per point in the beta_schedule according to a Metropolis-Hastings update. When beta is large the target distribution concentrates, at equilibrium, over ground states of the model. Samples are guaranteed to match the equilibrium for long 'smooth' beta_schedules, in practical operation the sampler can be used as a heuristic for equilibrium sampling, and optimization.
+
```

Comment:
* "Boltzmann sampling, or heuristic optimization" --> no comma 
* "In this implementation, the" or better "This implementation approaches equilibrium distribution at a target inverse temperature, beta, by performing updates at a sequence of increasing beta values, ``beta_schedule``, terminating ..."
* "target beta. In this implementation each spin is updated" --> "target beta. Each spin is updated..."
* "for long 'smooth' beta_schedules" here should be "beta schedules" 
* "schedules, in practical operation" --> needs a period here, not comma
* "used as a heuristic for equilibrium sampling, and optimization" --> "used as a heuristic for optimization and  equilibrium sampling"